metrics/access-logs/create-schema.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+set -x
+
+PROJECT=`gcloud config get-value project`
+
+# Must generally create in same location as GCS buckets, doesn't apply to US but still a good idea!
+# https://cloud.google.com/bigquery/docs/batch-loading-data#data-locations
+bq mk --location=US --dataset ${PROJECT}:kubernetes_public_logs
```

Comment:
@justinsb can we use this project and this dataset : https://github.com/kubernetes/k8s.io/pull/2031 ?